concourse/pipelines/dev-image-build.yaml

```diff
@@ -96,10 +100,34 @@ jobs:
       gcs_url: ((.:gcs-url))
       google_cloud_repo: "stable"
       build_date: ((.:build-date))
+  on_success:
+    task: success
+    file: guest-test-infra/concourse/tasks/publish-job-result.yaml
+    vars:
+      pipeline: "dev-image-build"
+      job: "build-debian-10-worker"
+      task: "publish-job-result"
+      result_state: "success"
+      start_timestamp: ((.:start-timestamp-ms))
+      metric_path: "concourse/job/duration"
+  on_failure:
+    task: failure
+    file: guest-test-infra/concourse/tasks/publish-job-result.yaml
+    vars:
+      pipeline: "dev-image-build"
+      job: "build-debian-10-worker"
+      task: "publish-job-result"
```

Comment:
this is not a variable in the task, i think it will give an error in use

Reply:
This didn't fail in dev pipeline testing, but the arg is no longer needed since it is directly specified anyways (moved into the task as per your suggestion).